Source/Thunder/Controller.cpp

```diff
@@ -1262,7 +1262,7 @@ namespace Plugin {
 
             while (it.Next() == true) {
                 auto const& entry = it.Current();
-                threads.push_back({ entry.Id.Value(), entry.Job.Value(), entry.Runs.Value() });
+                threads.push_back({ Id(entry.Id.Value()), entry.Job.Value(), entry.Runs.Value() });
```

Comment:
Doesn't JSON::ThreadId loose its purpose now? Likewise the parallel changes to JsonGenerator.

Reply:
done